MPin BLS461 Kangaroo test failing

The following error has been reported by Julian Feinauer (Apache IPMC member):
       TEST MPIN BLS461
         test MPin Kangaroo:
     AssertionError: expected 0 to equal 1111
      at Context.<anonymous> (test/test_MPIN.js:310:31)
      at processImmediate (internal/timers.js:443:21)
npm ERR! Test failed.  See above for more details.

Cannot reproduce on Windows or Ubuntu.  Likely need to confirm Julian's environment.

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,7 @@
 # To create the image execute:
 #     docker build -t mcjs:builder .
 #
-# To run tests:
+# To run the tests:
 #     docker run --rm mcjs:builder 
 #
 # To login to container:

--- a/README.md
+++ b/README.md
@@ -50,7 +50,7 @@ added New Hope, a post-quantum key exchange.
 
 ## Install and run  tests
 
-[Node.js](https://nodejs.org/en/) (minimum v6.9.5) and
+[Node.js](https://nodejs.org/en/) (maximum v8.0.0) and
 [npm](https://www.npmjs.com/) are required in order to build the library and
 run the tests. Install also the node.js modules required with the command
 
@@ -62,6 +62,20 @@ Run all the tests with the following command
 
 ```
 npm test
+```
+
+## Docker
+
+To create the image execute;
+
+```
+docker build -t mcjs:builder .
+```
+
+To run the tests;
+
+```
+docker run --rm mcjs:builder 
 ```
 
 ## Quick Start

